generate/generate_test.go

```diff
@@ -709,6 +709,144 @@ func TestGenerateConflictsBetweenGenerateTypes(t *testing.T) {
 				},
 			},
 		},
+		{
+			// https://github.com/terramate-io/terramate/issues/1260
+			// dotfiles should not be ignored if not inside a .tmskip
+			name: "regression test: generating dotfiles do not recreate all the time",
```

Comment:
I think this test is in the wrong function `TestGenerateConflictsBetweenGenerateTypes`. It doesn't affect the outcome, but it's mislabeled since the other test cases in this function are conflict-themed.

Reply:
yep, thanks for catching this!